src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -47,7 +47,7 @@
 }
 
 .pf-u-min-width {
-    --pf-u-min-width--MinWidth: 11ch;
```

Comment:
What was this needed for? I'm wondering why it was 18 to begin with - what components relied on it being 18?

Would it be better to inline the css for things like this so it is explicit?

I put in the .pf-u-max-width class at 26rem recently for the sources step and this is actually now making me think I should probably revisit that, as well.

Reply:
I've been using it to unify width of the description terms to align the lists into equal columns.

Reply:
Ah, that's right. I need to revisit how I did the layout on the sources step as I think I might have been relying on this as well... even if it turns out to be fine, I think that this is a good sign that we should avoid these global width changes when they are needed for specific components... Let's see if we can find a good inline solution so we can eliminate these global overrides.

Reply:
We can do this in a follow up PR, I'll create an issue so we don't forget.